chromium_src/chrome/browser/flags/android/chrome_feature_list.cc

```diff
@@ -17,6 +17,7 @@ const base::Feature* kBraveFeaturesExposedToJava[] = {
     &brave_rewards::features::kBraveRewards,
     &blink::features::kForceWebContentsDarkMode,
     &brave_wallet::features::kNativeBraveWalletFeature,
+    &brave_wallet::features::kBraveWalletSolanaFeature,
```

Comment:
should the CI be executed for all the platforms for this chromium change? skipping others because this is only related with android.

Reply:
no, we are good, that file is under Android section only in gn `if (is_android) {`
`chrome/browser/BUILD.gn:      "flags/android/chrome_feature_list.cc",`